tests/adam_test.cpp

```diff
@@ -217,6 +217,15 @@ TEST_CASE("PadamLogisticRegressionTest", "[AdamTest]")
   LogisticRegressionFunctionTest(optimizer, 0.003, 0.006);
 }
 
+/**
+ * Run RAdam on logistic regression and make sure the results are acceptable.
+ */
+TEST_CASE("RAdamLogisticRegressionTest", "[AdamTest]")
+{
+  RAdam optimizer;
+  LogisticRegressionFunctionTest(optimizer, 0.003, 0.006);
```

Comment:
might be useful to provide a brief comment on why these settings (0.003, 0.006) are used.